src/CommandingOfficers/Tech.java

```diff
@@ -219,38 +219,28 @@ public GameEventQueue getEvents(MapMaster gameMap)
       Set<XYCoord> dropLocs = new HashSet<XYCoord>();
       for( int i = 0; i < numDrops; ++i )
       {
-        TeleportEvent techDrop = generateDropEvent(gameMap, log);
+        TeleportEvent techDrop = generateDropEvent(gameMap, dropLocs, log);
         if( null != techDrop )
         {
           // Store the new event.
-          abilityEvents.add(techDrop);
-
-          // Place it on the map temporarily so we account for it in subsequent drops.
           dropLocs.add(techDrop.getEndPoint());
-          Unit u = techDrop.getUnit();
-          XYCoord loc = techDrop.getEndPoint();
-          gameMap.moveUnit(u, loc.xCoord, loc.yCoord);
```

Comment:
It seems you have agreed with my assessment. Works for me.

Reply:
Hah. Yes.